Examples/ObjectiveC/UserTrackingModesExample.m

```diff
@@ -3,23 +3,176 @@
 
 NSString *const MBXExampleUserTrackingModes = @"UserTrackingModesExample";
 
+#pragma mark - UIButton subclass
+
+// Subclass UIButton to create a custom user tracking mode button
+@interface UserLocationButton : UIButton
+@property (nonatomic) CAShapeLayer *arrow;
+@property (nonatomic) CGFloat buttonSize;
+@end
+
+@implementation UserLocationButton
+
+// Initializer to create the user tracking mode button
+- (instancetype)initWithButtonSize:(CGFloat)buttonSize {
+    if (self = [super init]) {
+        self.frame = CGRectMake(0, 0, buttonSize, buttonSize);
+        self.backgroundColor = [[UIColor whiteColor] colorWithAlphaComponent:0.8];
```

Comment:
Could you post an image/gif/movie/film/epic-masterpiece of how this button looks? I’m afraid white-on-slightly-off-white is difficult to read.